Makefile

```diff
@@ -194,6 +194,19 @@ antctl: $(ANTCTL_BINARIES)
 antctl-release:
 	@$(GO) build -o $(BINDIR)/$(ANTCTL_BINARY_NAME) $(GOFLAGS) -ldflags '-s -w $(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+.PHONY: add-copyright
+add-copyright: 
+	@echo "==> Installing addlicense <=="
+	@$(GO) install github.com/google/addlicense@latest
+ifeq ($(check), true)
```

Comment:
I don't think this is very user-friendly (having to define the check variable)

Reply:
maybe we can define two targets, one is `check-copyright`, one is `add-copyright`? or @bangqipropel  can change back to original code which was using `check-copyright --add` to add copy right, then developer can use `check-copyright` for checking only.

Reply:
I think `check-copyright` and `add-copyright` is better one, for now.
I think the key is here: we don't want to set var by the format "XX=YY" but the format "--XX" in the **make command**, but dealing with arg with dash is complicated in Makefile, (like [](https://stackoverflow.com/questions/2214575/passing-arguments-to-make-run)).  It will make the code much more complicated or let the command as `make add-copyright --  --check`,  two double-dash also seems like unfriendly to user, either.

Reply:
I am OK to add two targets since both targets.